### ISSUE TITLE
Separate all apps in 'Apps management' in two groups: system/non-system apps

### ISSUE THREAD

**Issue by vbagirov** [AdguardTeam/AdguardForAndroid#1110]
Original thread:
https://forum.adguard.com/index.php?threads/19881/#post-129709

**Comment by deLumren** [comment]
User 1439308 also suggested this idea.

**Comment by Bluscream** [comment]
D also

**Comment by TPS** [comment]
@All If you mean `System` as part of the OS, yes, absolutely!  But, if you mean `Builtin` as in part of the ROM, & therefore uninstallable outside of rooting, much less useful.  Bloatware ≠ System, please!

**Comment by vbagirov** [comment]
Other suggestions: 
- Only show apps that have network connection 
- Only show apps blocked by user

**Comment by TPS** [comment]
- Apps that have current `$app=` filters, user &/or filterlist
- *Un*filtered apps

**Comment by Revertron** [comment]
And one million more filters for that list, right?

**Comment by vbagirov** [comment]
@Revertron Those are just requests, we aren't obliged to add them all, but we can at least discuss if there are some good ones and a way to implement them.

**Comment by TPS** [comment]
[My comment from an entirely unrelated issue seems relevant here:](https://github.com/AdguardTeam/AdguardForAndroid/issues/108#issuecomment-149283877)
>I figure the point of a request like this, when users ask for the moon, start with a pebble & build from there — but don't forget to build! Moon-like timescale's perhaps ok. 😉

**Comment by TPS** [comment]
> And one million more filters for that list, right?

@Revertron _There's_ an idea: have some of these groups be dynamically-detected/-generated, i.e., only shown whenever AG can determine whether there _are_ unfiltered, user-blocked, _&c_, apps.  _(Thanks for the idea. 😇)_

**Comment by vbagirov** [comment]
Another request: add an option to sort alphabetically.

**Comment by TPS** [comment]
> add an option to sort alphabetically

@vbagirov That's https://github.com/AdguardTeam/AdguardForAndroid/issues/384#issuecomment-212609368.

**Comment by Bohdan-SUP** [comment]
User 1573869 suggests:
> ability to create custom groups and change settings for all apps in the group (not like the current way of all apps; modified vs unmodified)

**Comment by Bohdan-SUP** [comment]
Another request from VK:
>add an option to sort apps by the amount of saved data

**Comment by AlikhanAliev** [comment]
https://share.goabstract.com/94b4de0b-583b-4bed-9a70-cf90d46c6129

**Comment by ameshkov** [comment]
@AlikhanAliev actually, I'd like us to introduce more filters than just system/non-system. For instance, browsers/non-browsers, etc.

**Comment by AlikhanAliev** [comment]
https://share.goabstract.com/f20a127d-6522-4bb7-917a-318e305a60df

**Comment by AlikhanAliev** [comment]
https://share.goabstract.com/eed07d5a-03a5-4f8b-a59d-c81ad3d03e8c
new flow for "apps management"

**Comment by ameshkov** [comment]
Merged into https://github.com/AdguardTeam/AdguardForAndroid/issues/3379
